atd-vzv/src/views/summary/CrashesByTimeOfDay.js

```diff
@@ -35,90 +36,54 @@ const CrashesByTimeOfDay = () => {
     if (activeTab !== tab) setActiveTab(tab);
   };
 
+  const hourBlockArray = useMemo(
+    () => [...Array(24).keys()].map((hour) => moment({ hour }).format("hhA")),
+    []
+  );
+
   useEffect(() => {
     const dayOfWeekArray = moment.weekdaysShort();
 
-    const hourBlockArray = [
-      "12AM",
-      "01AM",
-      "02AM",
-      "03AM",
-      "04AM",
-      "05AM",
-      "06AM",
-      "07AM",
-      "08AM",
-      "09AM",
-      "10AM",
-      "11AM",
-      "12PM",
-      "01PM",
-      "02PM",
-      "03PM",
-      "04PM",
-      "05PM",
-      "06PM",
-      "07PM",
-      "08PM",
-      "09PM",
-      "10PM",
-      "11PM",
-    ];
-
-    let dataArray = [];
-
     const buildDataArray = () => {
-      dataArray = [];
-      hourBlockArray.forEach((hour) => {
-        let hourObject = {
-          key: hour,
-          data: [],
-        };
-        dayOfWeekArray.forEach((day) => {
-          let dayObject = {
-            key: day,
-            data: 0,
-          };
-          hourObject.data.push(dayObject);
-        });
-        hourObject.data.reverse();
-        dataArray.push(hourObject);
-      });
+      // This array holds weekday totals for each hour window within a day
+      // Heatmap expects array of weekday total objs to be reversed in order
```

Comment:
Maybe in the comment say the specific library in case we refactor the code later.

`// Reaviz Heatmap expects array of weekday total objs to be reversed in order`

Reply:
Added 👍 since we have discussed switching this out at some point.